parsers/GT.py

```diff
@@ -0,0 +1,90 @@
+# The arrow library is used to handle datetimes
+import arrow
+# The request library is used to fetch content through HTTP
+import requests
+# The pandas library is used to manipulate real time data
+import pandas as pd
+
+tz_gt = 'America/Guatemala'
+
+MAP_GENERATION = {
+    'coal': 'Turbina de Vapor',
+    'gas': 'Turbina de Gas',
+    'hydro': u'Hidroeléctrica',
+    'oil': 'Motor Reciprocante',
+    'solar': 'Fotovoltaica',
+    'wind': u'Eólico',
+    'geothermal': u'Geotérmica',
+}
+
+
+def fetch_production(country_code='GT', session=None):
+    #output frame
+    data = {
+        'countryCode': country_code,
+        'datetime': {},
+        'production': {},
+        'storage': {},
+        'source': 'www.amm.org.gt/',
+    }
+    #Get actual date
+    now = arrow.now(tz=tz_gt)
+    #Define url to request
+    formatted_date = now.format('DD/MM/YYYY')
+    url_init = 'http://wl.amm.org.gt/AMM_LectorDePotencias-AMM_GraficasWs-context-root/jersey/CargaPotencias/graficaAreaScada/' 
+    url = url_init + formatted_date
+    #Request and rearange in DF
+    r = requests.session()
+    response = r.get(url)
+    obj = response.json()
+    obj_df = pd.DataFrame(obj)
+    #Extract the corresponding hour
+    h = str(now.hour)
+    obj_h = obj_df[obj_df.hora == h]
+    #Fill datetime variable
+    data['datetime'] = now.replace(minute=0, second=0, microsecond=0).datetime   
```

Comment:
The response only have informations on the production per types and hours. Have a look at today's as an example:
http://wl.amm.org.gt/AMM_LectorDePotencias-AMM_GraficasWs-context-root/jersey/CargaPotencias/graficaAreaScada/03/08/2017